.github/workflows/testing.js.yml

```diff
@@ -176,20 +177,18 @@ jobs:
         run: rm -rf frank-runner/build/apache-tomcat-9.0.56/logs/catalina.out
 
       - name: Let Frank!Runner do its downloads
-        run: pnpm run installFrankRunnerEnv
-        working-directory: ladybug-frontend
+        run: cd frank-runner && ./env.sh > installFrankRunnerEnv.log
 
       - name: Start server using Frank!Runner in background
-        run: pnpm run startServer &
-        working-directory: ladybug-frontend
+        run: cd frank-runner/specials/ladybug && ./restart.sh -Dupdate.strategy=none > serverStartLog.log
```

Comment:
Same here

Reply:
Was geen goede reden voor. Is veranderd.